tests/unit/util/test_logging.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import logging.config
+from socket import gethostname
+
+import pytest
+
+from logprep.util.defaults import DEFAULT_LOG_CONFIG
+from logprep.util.logging import LogprepFormatter
+
+
+class TestLogprepFormatter:
+
+    def test_default_log_config(self):
+        logging.config.dictConfig(DEFAULT_LOG_CONFIG)
+        logger = logging.getLogger("test")
+        assert logger
```

Comment:
doesn't this assertion only check if the logger object exists, but not if the default configuration was actually applied?
```suggestion
        assert isinstance(logger.root.handlers[0].formatter, LogprepFormatter)
```